index.js

```diff
@@ -166,6 +169,11 @@ export const Rels = {
 	},
 	Users: {
 		canonicalUser: 'https://users.api.brightspace.com/rels/canonical-user'
+	},
+	UserProgress: {
+		Outcomes: {
+			activities: 'https://user-progress.api.brightspace.com/rels/outcome-activities'
```

Comment:
I'm not vehemently against this or anything, but we haven't namespaced rels beyond just the domain so far. Unless there's going to be another similar rel for `outcome-activities` that's outside the realm of `outcomes` (which seems unlikely?), this should probably just be `UserProgress.activities`.

My rationale here would be that the namespacing here matches the rel in a pretty 1:1 mapping, whereas `UserProgress.Outcomes.activities` I'd be more inclined to assign to something like https://user-progress.api.brightspace.com/rels/outcomes/activities`. No current plans to do so, but keeping that 1:1 mapping allows for simplifying how we generate these rels, and even someday making them auto-generated by the LMS. (e.g. some hook into the `FormatRel` call)

Reply:
This logic may not hold for the newly added feedback Rel (See updated changes)

Reply:
Changed